src/Command/PingWebsiteCommand.php

```diff
@@ -1,103 +1,113 @@
-use Doctrine\ORM\EntityManagerInterface;
-use App\Entity\Website;
+use Symfony\Component\HttpClient\CurlHttpClient;
 
 class PingWebsiteCommand extends Command
 {
     protected static $defaultName = 'app:website:ping';
-    
+
     private $em;
+    private $websiteRepository;
+    private $client;
 
-    public function __construct(EntityManagerInterface $em)
+    public function __construct(EntityManagerInterface $em, WebsiteRepository $websiteRepository)
     {
         parent::__construct();
         $this->em = $em;
+        $this->websiteRepository = $websiteRepository;
+        $this->client = new CurlHttpClient();
     }
 
     protected function configure()
     {
-        $this
-            ->setDescription('Ping one or more websites included in database.')
-            ->addArgument('websites', InputArgument::IS_ARRAY, 'Argument description')
-            ->addOption('all', 'a', InputOption::VALUE_NONE, 'Ping all websites to refresh status.')
-        ;
+        $this->setDescription('Ping one or more websites included in database.')->addArgument(
+                'websites',
+                InputArgument::IS_ARRAY,
+                'Argument description'
+            )->addOption(
+                'all',
+                'a',
+                InputOption::VALUE_NONE,
+                'Ping all websites to refresh status. This override websites argument.'
+            );
     }
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
+        $table = new Table($output);
         $io = new SymfonyStyle($input, $output);
-        $targettedWebsites = $input->getArgument('websites');
-        
-        $websiteRepo = $this->em->getRepository(Website::class);
-        
-        
-        
-        $websitesToPing = $websiteRepo->findBy(
-            ['name' => $targettedWebsites],
-        );
 
-        
-        
-        if ($websitesToPing) {
-            foreach($websitesToPing as $website) {
-                $io->note(sprintf('Pinging %s', $website));
-                $website = $this->callWebsite($website);
-                $this->em->persist($website);
-                $this->em->flush();
-                $output->writeln('');
-                $io->success('Status updated.');
-            }
-        } else if ($input->getOption('all')) {
-            $io->note('Requesting all websites in database.');
-            foreach($websiteRepo->findAll() as $website) {
-                $io->note(sprintf('Pinging %s', $website));
-                $website = $this->callWebsite($website);
-                $this->em->persist($website);
-                $this->em->flush();
-                $output->writeln('');
-                $io->success('Status updated.');
-            }
+        if ($input->getOption('all')) {
+            $io->title('Requesting all websites in database.');
+            $websitesToPing = $this->websiteRepository->findAll();
         } else {
```

Comment:
This should be a else if checking whether or not there is one or more website required. If this is not the case, return an error with help message or something.